app/views/home_page/show.html.slim

```diff
@@ -4,4 +4,5 @@ div.jumbotron
   hr.my-4
   p For now it's just a simple page
   p.lead
-    a class=['btn','btn-primary','btn-lg'] role="button" Learn more
+    a.btn.btn-primary.btn-lg[role="button"]
+      | Learn more
```

Comment:
@sinpai let's avoid pasting not localized text to the page. Imaging that you'll need a French translation? 😄 
Have to be moved to locale

Reply:
👍